.github/workflows/python-package-conda.yaml

```diff
@@ -2,20 +2,31 @@ name: build
 
 on: [push]
 
+env:
+  REPO_NAME: ${{ github.event.repository.name }}
+
 jobs:
-  pre-commit:
+  build-and-test:
```

Comment:
Why remove the pre-commit job and include it in `build-and-test`?

Reply:
Do you mean about renaming? or merging all steps? I merged them together since the same setup was running for pre-commit and test.